base/src/main/java/uk/ac/ebi/atlas/solr/analytics/query/AnalyticsQueryClient.java

```diff
@@ -36,41 +37,41 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(AnalyticsQueryClient.class);
     private final RestTemplate restTemplate;
     private final String solrBaseUrl;
-    private final Resource baselineFacetsQueryJSON;
-    private final Resource differentialFacetsQueryJSON;
+    private final Resource baselineFacetsQueryJson;
+    private final Resource differentialFacetsQueryJson;
     private final Resource experimentTypesQueryJson;
     private final Resource bioentityIdentifiersQueryJson;
 
     @Inject
     public AnalyticsQueryClient(RestTemplate restTemplate, @Qualifier("solrAnalyticsServerURL") String solrBaseUrl,
-                                @Value("classpath:/solr-queries/baseline.heatmap.pivot.query.json") Resource  baselineFacetsQueryJSON,
-                                @Value("classpath:/solr-queries/differential.facets.query.json") Resource differentialFacetsQueryJSON,
+                                @Value("classpath:/solr-queries/baseline.heatmap.pivot.query.json") Resource baselineFacetsQueryJson,
+                                @Value("classpath:/solr-queries/differential.facets.query.json") Resource differentialFacetsQueryJson,
                                 @Value("classpath:/solr-queries/experimentType.query.json") Resource experimentTypesQueryJson,
                                 @Value("classpath:/solr-queries/bioentityIdentifier.query.json") Resource bioentityIdentifiersQueryJson){
         this.restTemplate = restTemplate;
         this.solrBaseUrl = solrBaseUrl;
-        this.baselineFacetsQueryJSON = baselineFacetsQueryJSON;
-        this.differentialFacetsQueryJSON = differentialFacetsQueryJSON;
+        this.baselineFacetsQueryJson = baselineFacetsQueryJson;
+        this.differentialFacetsQueryJson = differentialFacetsQueryJson;
         this.experimentTypesQueryJson = experimentTypesQueryJson;
         this.bioentityIdentifiersQueryJson = bioentityIdentifiersQueryJson;
     }
 
     private String fetchResults(SolrQuery... qs ) {
         String result = "{}";
 
-        for(SolrQuery q: qs){
+        for (SolrQuery q: qs) {
             Stopwatch stopwatch = Stopwatch.createStarted();
-            LOGGER.debug("fetchResults {} took {} seconds", q, stopwatch.elapsed(TimeUnit.MILLISECONDS) / 1000D);
             result = fetchResponseAsString(MessageFormat.format("{0}query", solrBaseUrl), q);
             stopwatch.stop();
-            if(responseNonEmpty(result)){
+            LOGGER.debug("fetchResults {} took {} seconds", q, stopwatch.elapsed(TimeUnit.MILLISECONDS) / 1000D);
+            if (responseNonEmpty(result)) {
                 break;
             }
         }
         return result;
     }
 
-    boolean responseNonEmpty(String jsonFromSolr){
+    protected boolean responseNonEmpty(String jsonFromSolr) {
```

Comment:
I see no reason for you to have changed this and it's only used in fetchResults - make private?

Reply:
At some point I was reusing those methods in `AnalyticsClient`. At some point I think we’ll need to integrate the two classes. Thanks for spotting it.